docs/projects/blank/overview.md

```diff
@@ -1,3 +1,21 @@
-# Blank Template project
+# Blank Starter Project
```

Comment:
There's quite a few bits on here so have them all in one to make it easier to pop in:

```
# Blank Starter Project

The Blank Starter Project is a great starting point to working on the SpatialOS GDK for Unity if you want a clean slate.

Check out the [Blank Starter Project on Github](https://github.com/spatialos/gdk-for-unity-blank-project).

## What does it come with?

It has the minimum GDK feature set you need to start developing games for SpatialOS:

- A Unity project set up with the SpatialOS GDK for Unity.
- Sample [local and cloud deployment]({{urlRoot}}/contents/glossary#deploying) configurations.
- An empty snapshot.
- [Client-worker and server-worker]({{urlRoot}}/contents/glossary#worker) JSON [configuration files (SpatialOS documentation)](https://docs.improbable.io/reference/latest/shared/worker-configuration/bridge-config#worker-attribute-sets).
- Client-worker and server-worker [worker connectors]({{urlRoot}}/content/gameobject/creating-workers-with-workerconnector and [worker prefabs]({{urlRoot}}/content/gameobject/creating-workers-with-workerconnector#how-to-use-worker-prefabs).
- [Worker build configuration]({{urlRoot}}/content/build#build-your-workers).
- A set of Unity Scenes to use in development.

## Licensing

The Blank starter project is shared under the Improbable license; you can read more about what that entails in our [licensing]({{urlRoot}}/license) section.
```

Reply:
included these changes